www/include/configuration/configServers/help.php

```diff
@@ -4,7 +4,8 @@
 $help['ns_ip_address'] = dgettext("help", "IP address of the poller");
 $help['localhost'] = dgettext("help", "Whether the poller is local");
 $help['is_default'] = dgettext("help", "Main poller");
-$help['remote_id'] = dgettext("help", "Remote Server to which this server will be attached");
+$help['remote_id'] = dgettext("help", "Master Remote Server to which this server will be attached");
```

Comment:
please add the translated sentences or template

Reply:
the messages.pot file should be generated from all sources automatically